main.py

```
@@ -1,240 +1,195 @@
 import ell
+from ell import Field
 from openai import Client
 import os
 import argparse
+import subprocess
+from pydantic import BaseModel, Field
```

Ruff

`pydantic.BaseModel` imported but unused

Remove unused import

(F401)